doc/integrator/internationalization.rst

```diff
@@ -34,14 +34,22 @@ Build your application.
 
 The files to translate are:
 
-* ``demo/locale/<lang>/LC_MESSAGES/demo-client.po`` for ngeo client
-* ``demo/locale/<lang>/LC_MESSAGES/demo-server.po`` for the server part (should be empty for ngeo interfaces)
+* ``geoportal/<package>_geoportal/locale/<lang>/LC_MESSAGES/demo-client.po`` for ngeo client
+* ``geoportal/<package>_geoportal/locale/<lang>/LC_MESSAGES/demo-server.po`` for the server part (should be empty for ngeo interfaces)
 
 .. note::
 
    All the ``#, fuzzy`` strings should be verified and the line should be removed
    (if the line is not removed, the localisation will not be used).
 
+To update the po files you should run the specific target each time you change
```

Comment:
I think it would be clearer to say it the other way around:

Each time you change or add a new layer (mapfile, layer item in admin interface, etc.) or add a new feature, you should update the ``po`` files. You can do this by running a specific build target, as follows:

Reply:
Note that also conserne the print, the fulltextsearch.layername the application (javascript and HTML), the layer enumerations, the translatable metadata (disclamer), the editable layer (db structure, data for enums).